vault/audit_broker.go

```diff
@@ -10,6 +10,8 @@ import (
 	"sync"
 	"time"
 
+	"github.com/hashicorp/vault/helper/namespace"
+
```

Comment:
The extra blank line strikes again!

<img src="https://media2.giphy.com/media/9jVAv94PRzPoc/giphy.gif"/>

Reply:
> But the cat came back the very next day,
the cat came back, they thought he was a goner.
But the cat came back; it just couldn’t stay away.

Reply:
https://github.com/hashicorp/vault/pull/24248 fixes https://github.com/hashicorp/vault/pull/24238 😜